### PR TITLE
[DE-487](analytics) feat: refactor eple logic

### PR DESCRIPTION
**Pull request by cdelabre**
# DA PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [ ] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Fields have been snake_cased
- [ ] I have checked my modifications don't break downstream models
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I have updated the dag in cases of dependencies
- [ ] My code passes CI/CD tests
- [ ] I will post on slack review channel and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### CODE REVIEW

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql [hunk 1]
```diff
@@ -1,3 +1,12 @@
+WITH educational_institution_student_headcount AS (
+    SELECT
+        institution_id,
+        sum(headcount) as number_of_students,
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql:4]:
```suggestion
        sum(headcount) as total_students,
```

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+        avg(amount_per_student) as average_amount_per_student,
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql:5]: Ca serait peut être bien de préciser c'est un montant de quoi ici non ? Un montant pour les sorties scolaires c'est ça ?

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql [hunk 3]
```diff
@@ -15,5 +24,10 @@ select
...
         end,
         institution_departement_code
-    ) as institution_departement_code
+    ) as institution_departement_code,
+    number_of_students,
+    average_amount_per_student,
+    
 from {{ source('raw', 'applicative_database_educational_institution') }}
+left join educational_institution_student_headcount using (institution_id)
+where 1 = 1
```
Comment (carolinemestre-passculture) [inline orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql:33]: Pourquoi on ajoute le 1=1 ici ? Dans ce modèle on est pas censé avoir de filtre par la suite

File: jobs/etl_jobs/internal/cold-data/main.py [hunk 4]
```diff
@@ -31,7 +30,7 @@ def run():
             folder_name=folder_name,
             file_name=file_name,
             file_type=config.get("file_type"),
-            destination_dataset=RAW_DATASET,
+            destination_dataset=SEED_DATASET,
             destination_table=table,
             schema=config.get("schema"),
         )
```
Comment (LucileRainteau) [inline jobs/etl_jobs/internal/cold-data/main.py:36]: -- TODO : ajouter un appel GCS_to_GCS pour copier le fichier du bucket de prod vers le bucket de dev

File: orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql (hunk 2 above)
Comment (cdelabre) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql:5]: Oui, c'est le montant moyen par sortie scolaire pour un jeune (25€ / 30€), tu as une idée d'un nom que l'on pourrait mettre ?

Comment (carolinemestre-passculture) [reply orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__educational_institution.sql:5]: average_outing_budget_per_student ?

**Pull request merged**